src/datasets/model.rs

```diff
@@ -769,6 +769,12 @@ pub struct AplQueryResult {
     // NOTE: The following is copied from QueryResult. Maybe we should have a macro?
     /// The status of the query result.
     pub status: QueryStatus,
+    #[serde(
+        default,
+        alias = "datasetNames",
```

Comment:
Please do `#[serde(rename_all = "camelCase")]` on the struct instead

Reply:
done